### DISCUSSION TITLE
Vuejs 3 sample

### DISCUSSION THREAD

**Discussion by centratelemedia** [activej/activej#241] [General]
Helo,

can you provide sample integrated activej with vuejs 3?

TIA

**Comment by eduard-vasinskyi** [comment]
Hi, @centratelemedia

I am not that familiar with Vue.js to create an integration example. However, we have a few tutorials on how to integrate ActiveJ with React [[1](https://activej.io/tutorials/using-react), [2](https://activej.io/tutorials/todo-list-tutorial)]. Maybe you could use those tutorials as a reference for implementing Vue.js integration?
